test/workers/test_virtual.py

```diff
@@ -93,7 +93,7 @@ def test_recv_msg():
     # Test 2: get tensor back from alice
 
     # Create message: Get tensor from alice
-    msg = (MSGTYPE.OBJ_REQ, obj.id)
+    msg = sy.msg.Message(MSGTYPE.OBJ_REQ, obj.id)
```

Comment:
Why are there no other tests for messages?

Reply:
There is no new functionality for Message yet. I literally just replaced the tuple with a class with two attributes. Zillions of other tests already use it.

Reply:
There might be tests that are using it, but all of them are integration tests (meaning, that they use something that uses the class Message). It would be good to have a basic unit test as well.

Reply:
fair - i'll make a test :)

Reply:
i'll test serde - which is new

Reply:
groovy - added a test for message serde